prog-o-meter.py

```diff
@@ -145,8 +147,9 @@ def add_day(self):
         self.canvas.itemconfig(self.rectangle_list[self.days-1], fill = "green")
         update_days_file(self.filename, self.days)
         self.canvas.itemconfig(self.countdown_text, text = "".join(("You have ", str(self.days_remaining), " days left!\n\n", "If you code everyday, you will be done with this project on ", self.completion_date)))
-        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached 
-            self.add_day_button.config(state = "disabled")             
+        if self.days >=self.GOAL:        # Disable add_day_button if goal have been reached
+            self.add_day_button.config(state = "disabled")
+            Congratulations()        # Open congratulations window with link to share on Twitter

```

Comment:
Nice with a comment on this line, that explains the function shortly, since the function is not included directly in this .py file

Reply:
Sorry, I don't understand this comment. Did you want me to change something here, or leave it as-is?

Reply:
No, I was just appreciating the comment on line 152 :) But on line 150 we should update that comment to say something along "Disable add_day_button and congratulate user if goal have been reached"